sysutils/tmux/Portfile

```diff
@@ -9,8 +9,8 @@ if {${subport} eq ${name}} {
     conflicts       tmux-devel
 }
 subport tmux-devel {
-    github.setup    tmux tmux 52869ed182482c26163799a7215139f4d81b6fca
-    version         20160929-[string range ${github.version} 0 6]
+    github.setup    tmux tmux 5f662d91db658abbcd46d7a678c0250754be35e3
+    version         20170406-[string range ${github.version} 0 6]
     revision        1
```

Comment:
ok, I can remove the revision.

Until now, the committers always left it in, thus I never thought this was a problem.